runtime/src/main/java/org/corfudb/protocols/wireprotocol/IMetadata.java

```diff
@@ -206,6 +207,26 @@ default void setCheckpointedStreamStartLogAddress(Long startLogAddress) {
         getMetadataMap().put(CHECKPOINTED_STREAM_START_LOG_ADDRESS, startLogAddress);
     }
 
+    default void setPayloadCodecType(Codec.Type type) {
```

Comment:
Comments please. This is an interface.

Reply:
Done.

Reply:
I meant for all methods that have been added.